### ISSUE TITLE
identify the shapes for which to develop ShEX

### ISSUE THREAD

**Issue by egonw** [wikipathways/GPML2RDF#67]
The list (growing):

- [ ] Pathway
- [ ] DataNode
- [ ] GeneProduct
- [ ] Protein
- [ ] Metabolite
- [ ] Interaction
- [ ] DirectedInteraction
- [ ] TypedDirectedInteraction
- [ ]

**Comment by egonw** [comment]
I'm stuck (cc @andrawaag @nunogit):

![image](https://user-images.githubusercontent.com/26721/59202567-09ad7600-8b9d-11e9-8fea-c739ca749841.png)

**Comment by ericprud** [comment]
Yeah, the HTTPS miltia has seized control of the browsers. You're basically stuck setting up HTTPS everywhere in order to work in today's Web.

**Comment by egonw** [comment]
HTTP is working now.

**Comment by ericprud** [comment]
Meaning the browser is is no longer refusing to load HTTP resources? Is it because the rawgit image you loaded was from HTTP instead of HTTPS?

**Comment by egonw** [comment]
Interesting questions. Not sure. I always thought it was the HTTP/-S difference between the currently loaded HTML and the additional remote resources that mattered, not just between the additional remote resources.

Anyway, the one thing that changed is that the WikiPathways SPARQL endpoint now supports HTTPS too.

**Comment by ericprud** [comment]
> I also note the typo in my comment of two days ago, which should have said: "HTTPS is working now" :)

Ahh yes, that implies a different subject.

Re HTTP/S. This is a continuously-shifting battle-ground. As last I understood it, most browsers would refuse to fetch HTTPS from a page loaded with HTTP. At some point after that change, I believe rawgit changed to only serve HTTPS, which had the effect of requiring that all required resources be available by HTTPS. Then the squirrels arrived and chewed off my fingers and I swore off web development forever.

**Comment by egonw** [comment]
Some initial info here: https://github.com/egonw/shexWorkingBio/blob/master/index.md
